### DISCUSSION TITLE
Update copy on success page

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#140] [Feedback]
@gimyboya 

We are just thanking and guiding to explore more causes.  This is an opportunity to ask them to share about An-Nadaa. 
Ideally there should be a share button for social media but that can be a future feature.

(https://new.an-nadaa.com/success)

Proposed Copy:

 ### Jazakoum Allahu khayran. 
May Allah accept and increase you in goodness.

Please share with your friends and family and make du'aa for the success of this project.

[Explore more causes →](https://new.an-nadaa.com/causes)

**Comment by an-nadaa** [comment]
I forgot to add sharing capabilities to our individual cause page. We need to add that in the pipeline
